src/mlpack/tests/main_tests/preprocess_split_test.cpp

```diff
@@ -357,3 +357,26 @@ TEST_CASE_METHOD(
   REQUIRE(static_cast<uvec>(find(
       params.Get<arma::Mat<size_t>>("test_labels") == 2)).n_rows == 20);
 }
+
+/**
+ * Check that if training size is not zero.
```

Comment:
I don't understand this comment.

Reply:
I changed it to ``Check that if input dataset is not empty. Empty dataset can not be split.``